### PR TITLE
PIMS-1956: Single select filters

### PR DESCRIPTION
**Pull request by GrahamS-Quartech**
## 🎯 Summary

  
[PIMS-1956](https://apps.itsm.gov.bc.ca/jira/browse/PIMS-1956)

Added single select style filters where applicable.
This applies to
Properties:
* Agency
* Administrative Areas
* Classification

Projects:
* Agency 
* Status

Administrative Areas:
* Regional District

Agencies:
* Parent Agency

Users:
* Status
* Agency
* Role

For users, I thought it could also make sense to apply it to provider, but I'm not sure we can enumerate the options reliably. 

Please try filtering each of the affected columns to test. Also try filtering using the preset options in the top right and make sure those still work with these changes.

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

File: react-app/src/components/agencies/AgencyTable.tsx [hunk 1]
```diff
@@ -60,6 +70,8 @@ const AgencyTable = (props: IAgencyTable) => {
       field: 'ParentName',
       headerName: 'Parent Agency',
       flex: 1,
+      type: 'singleSelect',
+      valueOptions: agenciesForLookup,
```
Comment (dbarkowsky) [inline react-app/src/components/agencies/AgencyTable.tsx:74]: I think these should probably only be the parent agencies, not all agencies.

**Review by dbarkowsky — COMMENTED** [review]
All working really well. The only other column this could maybe be applied to is the Property Type.

I think there is one situation where the select filters don't populate values as expected.
In properties, changing the filter to a broad search like "Core" works as expected for the results but doesn't populate the filter model as expected.
image

File: react-app/src/components/agencies/AgencyTable.tsx (hunk 1 above)
Comment (GrahamS-Quartech) [reply react-app/src/components/agencies/AgencyTable.tsx:74]: Good catch, fixed this.

**Comment by GrahamS-Quartech** [comment]
> There is one situation where the select filters don't populate values as expected. In properties, changing the filter to a broad search like "Core" works as expected for the results but doesn't populate the filter model.

Fixed this, was just setting the value property slightly wrong.

**Comment by GrahamS-Quartech** [comment]
> All working really well. The only other column this could maybe be applied to is the Property Type.

Also added this.

**Review by dbarkowsky — APPROVED** [review]

**Pull request merged**